src/components/views/beacon/RoomCallBanner.tsx

```diff
@@ -35,7 +35,7 @@ const RoomCallBannerInner: React.FC<RoomCallBannerProps> = ({ roomId, call }) =>
                 action: Action.ViewRoom,
                 room_id: roomId,
                 view_call: true,
-                skipLobby: "shiftKey" in ev ? ev.shiftKey : false,
+                skipLobby: "shiftKey" in ev ? ev.shiftKey : undefined,
```

Comment:
should we just do `true` here? Otherwise it might be set to false specificly? Wouldnt the better fallback be the intent?

Reply:
Can it ever be false? I dont know what exactly is allowed in the webapi?